openmdao/docs/features/building_blocks/components/mux_demux_comps.rst

```diff
@@ -0,0 +1,74 @@
+
+.. _mux_demux_comp_feature:
+
+*********************
+MuxComp and DemuxComp
+*********************
+
+`DemuxComp` and `MuxComp` work together to break up inputs into multiple values (demux) or combine
+multiple inputs into a single value (mux).  This can be useful in situations where scalar outputs
+from multiple components need to be fed into a single vectorized component.
+
+`DemuxComp` takes a single input of arbitary shape (the size of at least one axis must be equal
+to `vec_size`).  In can then be broken along that axis, resulting in `vec_size` outputs.
+
+`MuxComp` combines two or more inputs into a single output by stacking them along an axis.
+
+MuxComp and DemuxComp Options
+-----------------------------
+
+These components have a single option, `vec_size`, which provides the number of inputs to be
+combined into a single output (for `MuxComp`) or the number of outputs into which an input is
+to be broken (for `DemuxComp`).  The default value of `vec_size` is 2.
+
+Adding Variables
+----------------
+
+A single `MuxComp` or `DemuxComp` can mux or demux multiple variables, so long as all variables
+are compatible with the given `vec_size`.  Variables are added via the `add_var` method.
+
+The axis along which the muxing/demuxing is to occur is given via the axis argument.  For DemuxComp,
+the specified axis must be be less than or equal to the number of dimensions in shape,
```

Comment:
"less than or equal to the number of dimensions in shape,"

Are you saying `axis` is Fortran style, starting at 1?  Seems like this should be just "less than" instead

Reply:
You can stack a 2D array on axes 0, 1, or 2.  Using axis=3 will raise an error.

Reply:
I think this needs better documentation then, for those who aren't familiar with the np.stack arguments.  Looks like it supports both 0 and -1 too.